tutorials/advanced/transforms_tutorial.py

```diff
@@ -303,6 +309,41 @@
 #
 #     trans = ax.get_xaxis_transform()
 #
+# .. _transforms-fig-scale-dpi:
+#
+# Plotting in physical units
+# ==========================
+#
+# Sometimes we want an object to be a certain physical size on the plot, in
+# which case the ``fig.dpi_scale_trans`` should be used.   The following
+# example draws a circle displaced one inch from the bottom-left corner that
+# is 0.5-inches in diameter.  Note that if you run this code with an
+# interactive window and resize, the circle stays in the same spot.
+
+fig, ax = plt.subplots(figsize=(3, 3))
+r = mpatches.Circle((1, 1), 1., transform=fig.dpi_scale_trans)
+fig.artists.append(r)
+plt.show()
+
+###############################################################################
+# Another use is putting a circle around a data point that is a certain size.
+# Though in this case if the figure is resized the circle doesn't follow the
+# data point.  This is a good example of using ``inverted``:
+fig, ax = plt.subplots()
+xdata, ydata = (0.2, 0.7), (0.5, 0.5)
+ax.plot(xdata, ydata, "o")
+ax.set_xlim((0, 1))
+# get the coordinates of the data point in inches from lower left corner:
+x, y = fig.dpi_scale_trans.inverted().transform(
+        ax.transData.transform((xdata[0], ydata[0])))
+# plot a circle around the point that is 50 points in diameter...
+circle = mpatches.Circle((x, y), 50/72, fill=None,
+        transform=fig.dpi_scale_trans)
```

Comment:
Not sure if this is a good example. If you want a circle with 50 point radius you'd just plot a `plot` with circular marker

    ax.plot(xdata, ydata, ls="", marker="o", ms=100,  mec="k", mfc="none")

Reply:
I think it would be good to explain `inverted()` somewhere. 

In the example here `inverted()` is not well explained. It's just there. Second, the example reads like this would be the way to go for placing some artist at a point in data coordinates. I doubt that is how one should do it. Essentially because of the drawback that it does not shift together with the data points.

To place some general path, like the ellipse here one could use
```
ell = mpatches.Ellipse((0, 0), 150/72., 130/72., angle=40)
p = ell.get_path()
verts = ell.get_patch_transform().transform(p.vertices)
codes = p.codes
p = mpath.Path(verts, codes)
ax.scatter(xdata, ydata, marker=p, s = (2*72)**2) 
```

Reply:
Ahhh:  

```python
mytrans =  fig.dpi_scale_trans + mtransforms.ScaledTranslation(xdata[0], ydata[0], ax.transData) 

print(mytrans)

ell = mpatches.Ellipse((0, 0), 150/72., 130/72., angle=40, transform=mytrans)
ax.add_patch(ell)
```

Reply:
That's a very good example. The problem is that `ScaledTranslation` and transform chaining are discussed later. So one should point/link to that in the text or maybe put the whole section more downwards?